### ISSUE TITLE
TestCudaCompiler Failure

### ISSUE THREAD

**Issue by kyleabeauchamp** [pandegroup/openmm#1624]
I have `OPENMM_CUDA_COMPILER` set properly, but `TestCudaCompiler` seems to fail:

```
:~$ cd ~/src/openmm/omm-build/
:~/src/openmm/omm-build$ ./TestCudaCompiler
Could not find nvcc.  Using runtime compiler, which may produce slower performance.  Set OPENMM_CUDA_COMPILER to specify where nvcc is located.
exception: Error compiling program: nvrtc: error: failed to load builtins
:~/src/openmm/omm-build$ $OPENMM_CUDA_COMPILER
nvcc warning : The 'compute_20', 'sm_20', and 'sm_21' architectures are deprecated, and may be removed in a future release (Use -Wno-deprecated-gpu-targets to suppress warning).
nvcc fatal   : No input files specified; use option --help for more information
:~/src/openmm/omm-build$ echo $OPENMM_CUDA_COMPILER
/usr/local/cuda-8.0/bin/nvcc
:~/src/openmm/omm-build$
```

**Comment by peastman** [comment]
The warning message is a bit misleading in this case.  TestCudaCompiler is a test for the CUDA runtime compiler, so it specifically disables using nvcc (by setting the CudaTempDirectory property to a nonexistant directory).  The real problem is this error:

```
exception: Error compiling program: nvrtc: error: failed to load builtins
```

It couldn't load the runtime compiler library (nvrtc).  I've also encountered that error on one computer, and haven't been able to figure out why.  In https://github.com/pandegroup/openmm/issues/1485 @jlmaccal reported running into it, but then said he'd fixed it and it was a library path issue.  Justin, can you give us more information about what it was?

Supposedly the poor performance with the runtime compiler is fixed in CUDA 8.0.  I haven't tested to verify that yet, but if it really is fixed, I think we can get rid of the warning message about performance.

**Comment by kyleabeauchamp** [comment]
This is solved on my system, either by #1629 or by me upgrading to Ubuntu 16.04, which is the officially supported version for CUDA 8.
